ui/cypress/integration/common/volume.js

```diff
@@ -68,11 +70,42 @@ Then(
       },
     );
 
+    // Check if the right side panel updates
+    cy.get('[data-cy="volume_detail_card_name"]').should(
+      'contain',
+      volumeNameSparseLoopDevice,
+    );
     // Check if all the labels are presents
-    cy.get('[data-cy=volume_label_name]').should('contain', volume_label_name);
-    cy.get('[data-cy=volume_label_value]').should(
+    cy.get('[data-cy="volume_label_name"]').should(
+      'contain',
+      volume_label_name,
+    );
+    cy.get('[data-cy="volume_label_value"]').should(
       'contain',
       volume_label_value,
     );
   },
 );
+
+And(
+  'I delete the volume by click on the delete volume button and confirm the deletion',
+  () => {
+    cy.get('[data-cy="delete_volume_button"]').click({ force: true });
```

Comment:
How does this work if "_the volume_" is not selected? What volume are we talking about?

Reply:
Actually after creating the volume "test-volume-sparse", I already implement the action "Click on this volume" in the code. 

So we are already in the volume "test-volume-sparse" detail on the right-side panel. 

But I encountered an error return by cypress here, that the `delete-volume-button` was covered by another element. That's why I add `.click({ force: true })` here.